package.json

```diff
@@ -64,9 +64,16 @@
     "webpack": "^5.90.3",
     "webpack-cli": "^5.1.4"
   },
+ bddsteps
```

Comment:
This might have come when resolving a conflict.

Reply:
Done